components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java

```diff
@@ -36,6 +36,9 @@
     public final static String MULTI_ATTRIBUTE_SEPARATOR = "MultiAttributeSeparator";
     public final static String MULTI_ATTRIBUTE_SEPARATOR_DEFAULT = ",,,";
 
+    public static final String XML_SIGNATURE_ALGORITHM_RSA_SHA1_URI = "http://www.w3.org/2000/09/xmldsig#rsa-sha1";
+    public static final String XML_SIGNATURE_ALGORITHM_RSA_SHA1 = "RSA with SHA1";
```

Comment:
Can remove this constant and use the URI constant everywhere including the combo box in the UI